packages/core/src/Card/stories/ExpandableCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import { css } from "@emotion/css";
+import {
+  HvCard,
+  HvCardContent,
+  HvCardHeader,
+  HvIconButton,
+  HvTypography,
+} from "@hitachivantara/uikit-react-core";
+import { Cloud, DropDownXS, User } from "@hitachivantara/uikit-react-icons";
+
+const classes = {
+  card: css({
+    width: 380,
+  }),
+  subheader: css({
+    display: "flex",
+    alignItems: "center",
+    "& > div": {
+      width: 16,
+      height: 16,
+    },
+  }),
+};
+
+export const ExpandableCard = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <HvCard className={classes.card} bgcolor="atmo1">
+      <HvCardHeader
+        title={<HvTypography variant="title4">Header Title</HvTypography>}
+        subheader={
+          <div className={classes.subheader}>
+            <User iconSize="XS" />
+            <HvTypography variant="caption2">Optional info</HvTypography>
+          </div>
+        }
+        icon={<Cloud />}
+      />
+      <HvCardContent
+        aria-expanded={isOpen}
+        style={{ display: isOpen ? "block" : "none" }}
+      >
+        <HvTypography>This is my hidden content</HvTypography>
+      </HvCardContent>
+
+      <div
+        style={{ display: "flex", justifyContent: "center", paddingBottom: 0 }}
+      >
+        <HvIconButton
+          title={isOpen ? "Collapse" : "Expand"}
+          onClick={() => setIsOpen(!isOpen)}
```

Comment:
Looking at [these docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded), I think we need to use the `aria-expanded` prop on the button and also use `aria-controls` (and set an `id` to the `HvCardContent`) 🙏